### PR TITLE
feat(frontend): adjust title for dapp explorer

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

Adjust title of DappExplorer according to spec.

# Changes

Explore Web3 Ecosystem => Explore apps

# Tests

Before
image
image

After
image
image

### CODE REVIEW

File: src/frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
 			"all_dapps": "All dapps",
 			"featured": "Featured",
 			"sign_in": "Sign-in to explore the ecosystem.",
-			"title": "Explore Web3 Ecosystem",
+			"title": "Explore apps",
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/i18n/en.json:92]: not uppercase since it is a title?

`Explore Apps`

**Review by AntonioVentilii-DFINITY — COMMENTED** [review]
My opinion: shouldn't be `Explore dApps` ? Or was it on purpose

File: src/frontend/src/lib/i18n/en.json (hunk 1 above)
Comment (loki344) [reply src/frontend/src/lib/i18n/en.json:92]: @StefanBerger-DFINITY 
ticket for reference: https://dfinity.atlassian.net/browse/OISY-276

Comment (StefanBerger-DFINITY) [reply src/frontend/src/lib/i18n/en.json:92]: Agree on the capitalization, if the UI guideline requires this for titles. So "Explore Apps"
About the Apps/Dapps: I got this exact text requested.

Comment (loki344) [reply src/frontend/src/lib/i18n/en.json]: adjusted

**Review by AntonioVentilii-DFINITY — APPROVED** [review]

**Pull request merged**